pkg/api/bzz.go

```diff
@@ -360,9 +360,10 @@ FETCH:
 					u.Path += "/"
 					redirectURL := u.String()
 
-					logger.Debug("bzz download: redirecting failed", "url", redirectURL, "error", err)
+					s.logger.Debug("bzz download: redirecting failed: not found", "url", redirectURL, "error", err)
+					s.logger.Error(nil, "bzz download: redirecting failed: not found")
```

Comment:
to me this debug message looks confusing because it indicates that redirection has failed, but there is no redirect operation.